models/appointment_resource_planning_model.py

```diff
@@ -0,0 +1,42 @@
+from dataclasses import dataclass, fields
+
+from models.database_base_model import DataBaseBase
+
+
+@dataclass
+class AppointmentResourcePlanning:
```

Comment:
This data class represents an Appointment so should this be just `Appointment`, reading `cati_appointment_resource_planning` in the code especially in `appointment_resource_planning_report.py` looks really confusing.